includes/push-ui.php

```diff
@@ -269,7 +270,7 @@ function ajax_push() {
 					$external_connection->log_sync( array( $remote_id => $_POST['postId'] ) );
 				} else {
 					$external_push_results[ (int) $connection['id'] ] = array(
-						'post_id' => (int) $remote_id,
+						'post_id' => $post_id,
```

Comment:
can we add an `error` attribute here with the (error) results from `$external_connection->push`? That will set us up for displaying the error on the front end.